views/js/qtiCreator/tpl/forms/response/responseForm.tpl

```diff
@@ -22,17 +22,19 @@
     </select>
 </div>
 
+{{#if textEntryInteraction}}
 <div class="panel">
     <label for="" class="has-icon">{{__ "Response base type"}}</label>
     <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
     <div class="tooltip-content">{{__ "Select the expected input value type that will define the way the response of your interaction will be processed."}}</div>
 
     <select name="listOfBaseType" class="select2" data-has-search="false">
         {{#each listOfBaseTypes}}
-        <option value="{{@key}}" ${@key === listOfBaseType ? selected : ''}>{{.}}</option>
+        <option value="{{value}}" {{#if selected}}selected="selected"{{/if}}>{{label}}</option>
```

Comment:
Does not make sense to bring yet one variable, when the real value is used. I would leave the previous condition, but with the proper source of data.

Reply:
I can use `@key` but I need selected and label, first for selected when you load a saved item and second if in the future we want a label different that real value

Reply:
Ok, I can imagine the situation, let's leave current solution, it is more reliable.